src/istgt_lu_disk.c

```diff
@@ -115,7 +116,8 @@ extern clockid_t clockid;
 			markedForReturn = 1;	\
 			goto macroname;	\
 		}	\
-		return 0;	\
+		markedForReturn = 2;	\
```

Comment:
enterblockingcall is called from many places. It doesn't seem that this error is handled everywhere.

Reply:
In above, can you change "IS_SPEC_READY" to "IS_SPEC_BUSY"? (not related to your PR)
current macro name is very confusing.